Lib/tkinter/__init__.py

```diff
@@ -2496,6 +2496,26 @@ def _loadtk(self):
             _default_root = self
         self.protocol("WM_DELETE_WINDOW", self.destroy)
 
+        # Fix for HiDPI screens on Windows
+        # CALL BEFORE ANY TK OPERATIONS!
+        # URL for arguments for the ...Awareness call below.
+        # https://msdn.microsoft.com/en-us/library/windows/desktop/dn280512(v=vs.85).aspx
+        if sys.platform == 'win32':
```

Comment:
In my humble opinion, can you guarantee that this code will not be called repeatedly? Maybe it only needs to be called once?

Reply:
Add a `print` statement to your code and just try the code below:

```py
from tkinter import messagebox

messagebox.showinfo("info", "test")
messagebox.showinfo("info", "test")
```

Then you'll find that it runs twice.